.github/workflows/docs.yaml

```diff
@@ -7,9 +7,7 @@ on:  # yamllint disable-line rule:truthy
 
 env:
   DOCS_REPO: "authzed/docs"
-  DOCS_BRANCH: "main"
-  GENERATED_DOCS_FILE: "docs/zed.md"
-  TARGET_DOCS_FILE: "pages/zed/zed.md"
+  TARGET_DOCS_FILE: "pages/spicedb/getting-started/installing-zed.mdx"
```

Comment:
This makes me wonder if we should reorganize this page on the docs side.

Reply:
Yeah, maybe...